Assets/SteamVR_Unity_Toolkit/Scripts/Controls/3D/VRTK_Chest.cs

```diff
@@ -0,0 +1,227 @@
+﻿namespace VRTK
+{
+    using UnityEngine;
+
+    public class VRTK_Chest : VRTK_Control
+    {
+        public enum Direction
+        {
+            autodetect, x, z // no support for y at this point in time
+        }
+
+        public Direction direction = Direction.autodetect;
+        private float min = 0f;
+        public float max = 160f;
```

Comment:
I wonder if this should be something more descriptive than max. Perhaps `maxRotation` ?